temel_bilgiler/giriş/web-geliştirmeye-giriş.md

```diff
@@ -0,0 +1,97 @@
+### Giriş
+
+Web geliştiricinin görevi nedir? Bir bakıma, websitesi yapma ve onun bakımından sorumludurlar.
+
+Web geliştiriciler genelde ürününü veya hizmetini web'e taşımaya çalışan müşteriler için çalışırlar. Çalışma genelde proje odaklıdır ve müşterinin ihtiyaçlarını son ürüne koordine eden bir ekip ile işbirliği yapmayı içerir. Müşteri bir teknoloji şirketi, bir organizasyon veya bir hükümet olabilir. Çalışma Frontend, Backend veya FullStack web geliştirme içerebilir.
```

Comment:
fix: Web geliştiriciler, genelde ürünlerini veya hizmetlerini internet ağına taşımaya çalışan müşteriler için çalışırlar.